native/c/zowex.cpp

```diff
@@ -53,6 +53,7 @@ int handle_job_release(ZCLIResult);
 int handle_console_issue(ZCLIResult);
 
 int handle_data_set_create_dsn(ZCLIResult);
+int handle_data_set_create_dsn_attr(ZCLIResult);
```

Comment:
Are we creating a new function and keeping the original? 
Or is the intention of the PR to modify the original?